draft-ietf-scitt-architecture.md

```diff
@@ -357,10 +354,11 @@ The terms "header", "payload", and "to-be-signed bytes" are defined in {{-COSE}}
 
 The term "claim" is defined in {{RFC8392}}.
 
-Append-only Log (Ledger):
+Append-only Log:
+
+A sequence of Signed Statements captured by a Verifiable Data Structure.
```

Comment:
```suggestion
Registered Sequence of Signed Statements:

A sequence of Signed Statements submitted by Clients that meet applicable registration policies and are captured by a Verifiable Data Structure.
```

Proposal for a term to designate the sequence of signed statements committed to the VDS, as opposed to the VDS itself, and hopefully move away from the controversial "Append-only Log". It's useful to have a term for this, when discussing audit.

Reply:
Append-only VDS?

Reply:
Resolving, with Issue #352